comeet/UI/RoomsList/RoomsListViewModel.swift

```diff
@@ -159,16 +159,22 @@ class RoomsListViewModel : BaseViewModel {
         }
         return availableRooms()[index]
     }
-    
+
     func startTime(value: CGFloat) -> String {
+        let count = availableRooms().count
         startDate = Date.dateFrom(sliderValue: Float(value), date: startDate)
-        reloadBinding?()
+        if count != availableRooms().count {
+            reloadBinding?()
+        }
         return startDate.displayStringHourMinute()
     }
     
     func endTime(value: CGFloat) -> String {
+        let count = availableRooms().count
         endDate = Date.dateFrom(sliderValue: Float(value), date: endDate)
-        reloadBinding?()
+        if count != availableRooms().count {
```

Comment:
There's a case where one room becomes unavailable and another becomes available where the count of rooms doesn't change but the list does.